packages/gestalt/src/Link.js

```diff
@@ -17,6 +17,7 @@ type TapEvent =
 type Props = {|
   accessibilitySelected?: boolean,
   children?: React.Node,
+  forwardedRef?: React.Ref<'a'>,
```

Comment:
should we do `React.Ref<HTMLAnchorElement>` for consistency? is there any difference?

Reply:
@jackhsu978 neither of these work:

![image](https://user-images.githubusercontent.com/127199/87458670-2a75a580-c5bf-11ea-96ca-08cc7c3a6dd8.png)
![image](https://user-images.githubusercontent.com/127199/87458740-41b49300-c5bf-11ea-8ac5-5d97b523c843.png)

In other places we have also used `React.Ref<'input'>` and `React.Ref<'div'>`